linkis-computation-governance/linkis-engineconn-manager/linkis-engineconn-manager-server/src/main/scala/org/apache/linkis/ecm/server/service/impl/LinuxProcessEngineConnLaunchService.scala

```diff
@@ -39,14 +39,14 @@ class LinuxProcessEngineConnLaunchService extends ProcessEngineConnLaunchService
       engineConnBuildRequest: EngineConnBuildRequest,
       sender: Sender
   ): EngineNode = {
-    Sender.getSender(ENGINECONN_PLUGIN_SPRING_NAME).ask(engineConnBuildRequest) match {
```

Comment:
Is there no need to request linkisManager here, the final EngineConnLaunchRequest sent by linkismanager is fine